schema_salad/main.py

```diff
@@ -79,11 +77,18 @@ def main(argsl=None):  # type: (List[str]) -> int
     exgroup.add_argument("--debug", action="store_true",
                          help="Print even more logging")
 
-    parser.add_argument("schema", type=str)
+    parser.add_argument("schema", type=str, nargs="?", default=None)
     parser.add_argument("document", type=str, nargs="?", default=None)
+    parser.add_argument("--version", "-v", action="store_true",
+                        help="Print version", default=None)
+
 
     args = parser.parse_args(argsl)
 
+    if args.version is None and args.schema is None:
```

Comment:
since ``schema`` has been made optional, it is explicitly needed to check if it is passed.